# goal
string posture_type
bool reception_detection
bool context_aware
geometry_msgs/PoseStamped person_pose
---
# result
bool success
---
# feedback
string current_state
string message
